examples/dir-extract.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import time
+import traceback
+from pathlib import Path
+
+from indico import IndicoClient
+from indico.config import IndicoConfig
+from indico.queries.documents import DocumentExtraction
+
+
+def ocr_files(target_files):
+
+    my_config = IndicoConfig(
+        host='dev.indico.io',
+        api_token_path=Path(__file__).parent / 'indico_api_token.txt'
+    )
+
+    client = IndicoClient(config=my_config)
+
+    jobs = []
+    failed = []
+
+    for src_path in target_files:
+        finfo = src_path.stat()
+
+        if finfo.st_size > 0:
+            start_time = time.time()
+            job = client.call(DocumentExtraction(files=[src_path], json_config='{"preset": "simple"}'))
+            upload_time = time.time() - start_time
+            if job is not None:
+                jobs.append(job)
+                print(f"({len(jobs)}) Queued OCR on {src_path.parent.name}/{src_path.name} ({(finfo.st_size / 1000):.2f}kb in {upload_time:.2f}s)")
+            else:
+                print(f"ERROR - Upload failed on {src_path.parent.name}/{src_path.name} - skipping")
+                failed.append(src_path)
+        else:
+            print(f"ERROR - Zero length file {src_path.parent.name}/{src_path.name} - skipping")
+            failed.append(src_path)
+    
+    total_jobs = len(jobs)
+    n_finished = 0
+    print(f"\nQueued {total_jobs} total files for OCR")
+    print("=" * 40)
+
+    for path, jobs in zip(target_files, jobs):
```

Comment:
this has to be rewriten with the new JobsStatus query

Reply:
Example removed